etc/docker/Dockerfile

```diff
@@ -28,23 +28,59 @@
 FROM ubuntu
 
 ENV DEBIAN_FRONTEND noninteractive
+ENV PLATFORM ubuntu
 ENV RELEASE 1
 ENV NAT64 1
 ENV DNS64 1
 
 COPY . /app
-
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y git lsb-core sudo
+# Required during build or run
+ENV OTBR_DOCKER_REQS        \
```

Comment:
nit: I don't think we need to align `\`, according to https://google.github.io/styleguide/shell.xml?showone=Pipelines#Pipelines